example/grpc-example/main.go

```diff
@@ -0,0 +1,149 @@
+// (c) Copyright IBM Corp. 2021
+// (c) Copyright Instana Inc. 2016
+
+// +build go1.10
+
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"net"
+
+	"github.com/opentracing/opentracing-go/ext"
+
+	"github.com/instana/go-sensor/example/grpc-example/pb"
+
+	instana "github.com/instana/go-sensor"
+	"github.com/instana/go-sensor/instrumentation/instagrpc"
+	"google.golang.org/grpc"
+)
+
+const (
+	defaultPort    = ":43210"
+	defaultAddress = "localhost"
+	testMessage    = "Hi!"
+)
+
+func main() {
+	// `port` will be used by an example
```

Comment:
Let's omit the comments that are not related to instrumentation with Instana

Reply:
Makes sense, I will change it.

Reply:
This is done.